nowcasting_dataset/data_sources/fake.py

```diff
@@ -118,6 +119,29 @@ def satellite_fake(
     return Satellite(xr_dataset)
 
 
+def optical_flow_fake(
+    batch_size=32,
+    seq_length_5=19,
+    satellite_image_size_pixels=64,
+    number_satellite_channels=7,
+) -> OpticalFlow:
+    """ Create fake data """
+    # make batch of arrays
+    xr_arrays = [
+        create_image_array(
+            seq_length_5=seq_length_5,
+            image_size_pixels=satellite_image_size_pixels,
+            number_channels=number_satellite_channels,
```

Comment:
you may have to merge from 'main' as there may be a conflict here

Reply:
Done!